pandas/tests/series/methods/test_replace.py

```diff
@@ -36,6 +36,15 @@ def test_replace_explicit_none(self):
         assert expected.iloc[-1] is None
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("to_replace, value", [(np.nan, pd.NA), (np.nan, None)])
```

Comment:
same comments as above. consider just using the null_fixture which has all the nans

Reply:
`nulls_fixture` was a good solution. Implemented.